src/main/java/in/bloomapp/userservice/UserManager.java

```diff
@@ -0,0 +1,78 @@
+package in.bloomapp.userservice;
+
+import in.bloomapp.dao1.UserManagerDAO;
+import in.bloomapp.exception.DBException;
+import in.bloomapp.exception.InvalidInputException;
+import in.bloomapp.exception.UserServiceException;
+import in.bloomapp.model.User;
+import in.bloomapp.util.IsValid;
+import in.bloomapp.validator.UserValidator;
+
+public class UserManager {
+
+	private UserManager() {
+		
+	}
+	/**
+	 * Validates all constrains adds the user to the Dao
+	 * @param userName
+	 * @param password
+	 * @param email
+	 * @param mobileNo
+	 * @param address
+	 * @return
+	 * @throws UserServiceException
+	 * @throws DBException
+	 */
+	public static boolean addUser(String userName,String password,String email,String mobileNo,String address) throws UserServiceException, DBException {
+
+		boolean status = false;
+		// checks for blank spaces
+		try {
+			IsValid.isValidString(userName);
+			IsValid.isCharAllowed(userName);
+			IsValid.isValidPassword(password);
+			IsValid.isValidMobileNo(mobileNo);
+			IsValid.isValidEmail(email);
+			IsValid.isValidString(address);
+			User newUser = new User(userName,password,email,mobileNo,address);
+			UserManagerDAO.save(newUser);
+			status = true;
+			return status;
+		} catch (InvalidInputException e) {
```

Comment:
instead of catching exceptions in service u can use throws here at the top and let servlet catch those exceptions